pkg/daemon/server/service/rater/helper.go

```diff
@@ -17,13 +17,17 @@ limitations under the License.
 package rater
 
 import (
+	"math"
 	"time"
 
 	sharedqueue "github.com/numaproj/numaflow/pkg/shared/queue"
 )
 
 const (
 	indexNotFound = -1
+	// rateNotAvailable is returned when the processing rate cannot be derived from the currently
+	// available pod data, a negative min is returned to indicate this.
+	rateNotAvailable = float64(math.MinInt)
```

Comment:
please make sure that the -ve val returned [by ud-source gRPC](https://github.com/numaproj/numaflow/blob/main/pkg/apis/proto/source/v1/source.proto#L139-L141) is set as `rateNotAvailable`